views/app-welcome.hbs

```diff
@@ -0,0 +1 @@
+<h2><a href="/compliments">Give me a compliment</a></h2>
```

Comment:
This link is a reference to a route that contains a listing of all compliments ("/compliments").  It doesn't sound appropriate for the caption "Give me **a** compliment."